app/views/pieces/_form.html.haml

```diff
@@ -1,15 +1,15 @@
 = form_for(@piece) do |f|
-  %div
+  %div.review-form-title
     = f.label :url
     = f.text_field :url, placeholder: "YouTube URL"
 
-  %div
+  %div.review-form-title
     = f.label :title
     = f.text_field :title, placeholder: "Title"
 
-  %div
+  %div.review-form-title
```

Comment:
`%div.review-form-title` can be written as `.review-form-title` since `%div` is implicit